app/src/test/java/org/p2p/wallet/auth/web3authsdk/mapper/Web3AuthClientMapperTest.kt

```diff
@@ -0,0 +1,35 @@
+package org.p2p.wallet.auth.web3authsdk.mapper
+
+import assertk.all
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNotNull
+import assertk.assertions.prop
+import com.google.gson.Gson
+import org.junit.Test
+import org.p2p.wallet.auth.gateway.repository.model.GatewayOnboardingMetadata
+
+class Web3AuthClientMapperTest {
+
+    @Test
+    fun `test F iOS metadata string wrapped fix`() {
```

Comment:
nice

Reply:
i thought about `superfluous` but it is the same